testing/tests/DevExpress.exporter/jspdfParts/jspdf.printInfoCalculator.tests.js

```diff
@@ -0,0 +1,324 @@
+import { ExportPrintInfoCalculator } from 'exporter/jspdf/export_print_info_calculator.js';
+import { JSPdfPrintInfoCalculatorTestHelper } from './jspdfTestHelper.js';
+
+let helper;
+
+const JSPdfPrintInfoCalculatorTests = {
+    runTests(moduleConfig) {
+        helper = new JSPdfPrintInfoCalculatorTestHelper(this.jsPDFDocument);
+
+        QUnit.module('Print Info Calculator', moduleConfig, () => {
+            QUnit.test('Empty grid', function(assert) {
+                const done = assert.async();
+
+                const options = {
+                    jsPDFDocument: this.jsPDFDocument,
+                    cells: []
+                };
+                const calculatedOptions = ExportPrintInfoCalculator.calculate(options);
+                assert.strictEqual(calculatedOptions.columnStyles[0], undefined, 'columnStyles[0]');
+
+                done();
+            });
+
+            QUnit.test('1 column, 1 rows', function(assert) {
+                const done = assert.async();
+
+                const options = {
+                    jsPDFDocument: this.jsPDFDocument,
+                    cells: [[{ content: 'text' }]]
+                };
+                const calculatedOptions = ExportPrintInfoCalculator.calculate(options);
+                helper.checkColumnWidths([calculatedOptions.tableWidth], calculatedOptions);
+
+                done();
+            });
+
+            QUnit.test('2 columns, 1 rows', function(assert) {
+                const done = assert.async();
+
+                const options = {
+                    jsPDFDocument: this.jsPDFDocument,
+                    cells: [[{ content: 'text_1' }, { content: 'text_2' }]]
+                };
+                const calculatedOptions = ExportPrintInfoCalculator.calculate(options);
+
+                const columnWidth = calculatedOptions.tableWidth / 2;
```

Comment:
It's better to set explicit tableWidth in tests and add few additional tests to check how it is automatically calculated.